src/libraries/System.Text.RegularExpressions/ref/System.Text.RegularExpressions.cs

```diff
@@ -4,6 +4,8 @@
 // Changes to this file must follow the https://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
 
+using System.Runtime.Serialization;
```

Comment:
It was added automatically, I think. If you think it's safe to delete, please do, as in on mobile and cannot make changes atm, and the deadline is closing in ;)

Reply:
Added automatically by the reference source generator tool? I doubt that. More likely you edited this file manually in VS?

Reply:
@stephentoub, yes, I did ;). The editor must've added it and I missed it. If it's not supposed to be there, I've no problem with it being removed :).

Reply:
I cannot seem to do this from mobile, can you?

Reply:
@stephentoub: I tried removing it, but then I cannot build anymore. I think it's necessary after all (and indeed, it is referenced in the file). Or there another approach I'm not aware of?

Reply:
You need to fully qualify all the type names. As I stated at the beginning, we don't use usings in refs

Reply:
It's just because you did the edit by hand not using the tool
https://github.com/dotnet/runtime/blob/7f99e88afb6f281146f0e794b735a24e8cac2ce3/docs/coding-guidelines/updating-ref-source.md#for-most-assemblies-within-libraries

I just ran it, and the diffs seem harmless to me.. They will vanish next time someone runs the tool.